resources/views/home.blade.php

```diff
@@ -0,0 +1,35 @@
+@extends('layouts.app')
+
+@section('content')
+<div class="container">
+    <div class="row justify-content-center">
+        <div class="col-md-8">
+            <div class="card">
+                <div class="card-header">{{ __('Dashboard') }}</div>
+
+                <div class="card-body">
+                    @if (session('status'))
+                        <div class="alert alert-success" role="alert">
+                            {{ session('status') }}
+                        </div>
+                    @endif
+
+                    {{ __('You are logged in!') }}
+                </div>
+            </div>
+            <br/>
+            <div class="card">
+                <div class="card-header">Do you wanna upload/change your avatar?</div>
+                <div class="card-body">
+                    <x-alert />
+                    <form action="/upload" method="post" enctype="multipart/form-data">
```

Comment:
Is better to write real laravel routes, like `route('user.upload')` (you must assign the name to the route)

Reply:
Have forgotten to change that 👍